### PR TITLE
add pycudasirecon

### PR DESCRIPTION
**Pull request by tlambert03**
python wrapper for the recently added [cudasirecon](https://github.com/conda-forge/cudasirecon-feedstock)

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by tlambert03** [comment]
@conda-forge/help-python, this is ready for review.  Thanks!

File: recipes/pycudasirecon/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    - python >=3.7
+    - numpy
+    - pydantic
+    - cudasirecon
+    - tifffile
+    - typing_extensions
+
+test:
+  imports:
+    - pycudasirecon
```
Comment (chrisburr) [inline recipes/pycudasirecon/meta.yaml:31]:
```suggestion
    - pycudasirecon
    - pip check
  requires:
    - pip
```

Comment (tlambert03) [reply recipes/pycudasirecon/meta.yaml:31]: Merged, but broke it.  Should be a command no?

File: recipes/pycudasirecon/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    - pip check
```
Comment (chrisburr) [inline recipes/pycudasirecon/meta.yaml:32]:
```suggestion
  commands:
    - pip check
```

File: recipes/pycudasirecon/meta.yaml (hunk 1 above)
Comment (chrisburr) [reply recipes/pycudasirecon/meta.yaml:31]: Yup sorry about that 😅

Comment (tlambert03) [reply recipes/pycudasirecon/meta.yaml:31]: no worries.  thanks for the tip!

**Comment by tlambert03** [comment]
ready to go @chrisburr, @conda-forge/help-python

**Comment by tlambert03** [comment]
thanks again!

**Pull request merged**